cloudbridge/interfaces/resources.py

```diff
@@ -572,6 +572,26 @@ def reboot(self):
         """
         pass
 
+    @abstractmethod
+    def start(self):
+        """
+        Start this instance (using the cloud middleware API)
+
+        :rtype: ``bool``
+        :return: ``True`` if the starting was successful; ``False`` otherwise.
+        """
+        pass
+
+    @abstractmethod
+    def stop(self):
+        """
+        Stop this instance (using the cloud middleware API)
+
+        :rtype: ``bool``
+        :return: ``True`` if the stopping was successful; ``False`` otherwise.
```

Comment:
The functions below don't have return statements

Reply:
(and yes that seems to be wrong with the existing reboot as well)

Reply:
Sorry I didn't understand this part can you please elobrate. My understanding is since its just a interface, there is no point in adding return statement (that's why pass is added in every method). If i have to add return statement, it would be hard coding the return value which doesn't make sense.

Reply:
The “pass” in the interface is fine, you can think of it as defining the abstract skeleton of the functions. However, you’ve declared here that this function will return a Boolean, which will be True if the operation succeeded and False if it didn’t. You should have your implementation if the functions below (in ‘aws/resources.py’) match your abstract definitions in the interface. I.e the functions should return true if the operation was successful, and false if not. Currently they return nothing, so ‘st = inst.stop()’ never results in ‘st’ being a Boolean as expected by the interface

Reply:
hey @almahmoud did you get a chance to review the latest commit?